src/types.rs

```diff
@@ -0,0 +1,20 @@
+//! Provides primitive fixed size hash types.
+
+// Constructed via `fixed_hash` macro.
+
+construct_hash!(H32, 4);
+construct_hash!(H64, 8);
+construct_hash!(H128, 16);
+construct_hash!(H160, 20);
+construct_hash!(H256, 32);
+construct_hash!(H264, 33);
+construct_hash!(H512, 64);
+construct_hash!(H1024, 128);
+construct_hash!(H2048, 256);
```

Comment:
probably only H160 and H256 needed?

Reply:
You are probably right, I just cloned the definitions from `parity-hash`.

Reply:
I am especially suspicious about the `H264` one. :D

Reply:
oh, it is used for some signatures - it is ok, just not needed here